static/bidder-info/adot.yaml

```diff
@@ -12,3 +12,8 @@ capabilities:
       - banner
       - video
       - native
+userSync:
+  redirect:
+    url: "https://sync.adotmob.com/cookie/pbs?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&r={{.RedirectURL}}"
```

Comment:
I tested this syncer and found that a `gdpr_consent` is always added to the end of the redirect url. The adapter should not change any part of the redirect url except for the user macro replacement.

Reply:
I updated the syncer and it should now always use the redirect url in the query param with no modification except for the uid macro. Can you tell me if it looks good to you ?